### DISCUSSION TITLE
Why Are there race restrictions?

### DISCUSSION THREAD

**Discussion by tenaz123** [DarkCastleMUD/DarkCastle#239] [Q&A]
Aside from balance issues, why should any race be restricted from being a class they want to be?  In fantasy novels, the most compelling characters are often odd-ball race/class combos.

Also, it never made sense that an elf couldn't be an Anti.

If someone wants to be a giant thief or a pixie barb, then let them at it.  The easiest balance fix is to remove thief from the harpoon so they don't get uber dual stab as a giant when they Powerwield.  They could at best use it to wield 2 heavier one-handed weapons.

**Comment by TehDreadPirateRoberts** [comment]
I agree with this.
